SwifterDemoiOS/SceneDelegate.swift

```diff
@@ -2,30 +2,27 @@
 //  SceneDelegate.swift
 //  SwifterDemoiOS
 //
-//  Created by noppe on 2019/09/23.
+//  Created by Andy Liang on 2019-09-22.
 //  Copyright © 2019 Matt Donnelly. All rights reserved.
 //
 
 import UIKit
 import SwifteriOS
 
+@available(iOS 13.0, *)
 class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     var window: UIWindow?
 
-    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
-        
-    }
-    
     func scene(_ scene: UIScene, openURLContexts URLContexts: Set<UIOpenURLContext>) {
-        for context in URLContexts {
```

Comment:
Similarly here, somewhere that has a flag like `let isUsingSSO: Bool = false` , and something like the following. It's also probably a good idea to leave a comment about how this works, so that users will know which to use (and to only implement one over the other).
```
if isUsingSSO {
  // ... SSO Callback
} else {
  // ... Web
}
```